tests/internal/scrollable-container-test.html

```diff
@@ -0,0 +1,88 @@
+<!doctype html>
+<html lang=en>
+  <meta charset="utf-8">
+  <title>Spatnav sanity check</title>
+  <script src="../../polyfill/spatial-navigation-polyfill.js"></script>
+  </script>
+  <link rel="stylesheet" href="../../demo/sample/spatnav-style.css">
+  <link rel="stylesheet" href="test.css">
+  <script src="test.js"></script>
+  <style>
+  #d {
```

Comment:
where is targeted element? I couldn't find element with id="d"

Reply:
I guess this is a copy-paste error. I removed this declaration via https://github.com/WICG/spatial-navigation/commit/a924f016d59ef2a54f25c74a60523b822b5c1cf0